### ISSUE TITLE
BW mode and slider

### ISSUE THREAD

**Issue by psstoyanov** [PNDeb/pinenote-gnome-extension#12]
## Info

Move the black and white slider bellow BW Mode - it would be easier for users to understand that they are connected.

![Screenshot From 2024-12-10 12-44-00](https://github.com/user-attachments/assets/4d375ff0-a9e4-4e54-b0f4-28c5e7f5973c)

**Comment by PeterNerlich** [comment]
Does it even work? I Tried it yesterday while having a simple test image with a gradient up, and I couldn't see any difference

**Comment by psstoyanov** [comment]
Yup, it does work.
Depending on the contents, you may have to go near an extreme.


https://github.com/user-attachments/assets/f85caf36-01bc-44f1-b1ab-553ac1fb4387

**Comment by PeterNerlich** [comment]
Huh, curious. Turns out it does work for me after all, but only spontaneously, most often only after closing the popup menu.

https://github.com/user-attachments/assets/b8c991d2-0234-46c7-ad55-f365c486a7c1

**Comment by psstoyanov** [comment]
Looks like the changes are applied once you release finger from the slider, not while the finger is on it or it's actively changing.

**Comment by PeterNerlich** [comment]
Yes, but not always. Look at the tap at second ~14 in my video, it is a big difference that isn't reflected on the screen at all. It was like this the whole time I tried yesterday, and I didn't notice it updating when closing the popup menu
